src/components/Utils/getLocationsForSelect.js

```diff
@@ -1,6 +1,4 @@
-import { get } from 'lodash';
-
-const getLocationsForSelect = (resources) => get(resources, 'locations.records', []).map(({ name, id, code }) => ({
```

Comment:
What is the purpose of this change? Within it you have to duplicate code of getting `_.get(resources, 'locations.records', [])` in every place where you want to use `getLocationsForSelect`

Reply:
Rolled back. I reused this function in `ReceivigList`